be/src/storage/page_cache.h

```diff
@@ -38,6 +38,7 @@
 #include <string>
 #include <utility>
 
+#include "cache/object_cache/object_cache.h"
```

Comment:
`#include "util/lru_cache.h"` is useless?

Reply:
Yes, it can be removed, i will remove it next pr.